common/services/elasticsearch.ts

```diff
@@ -1,21 +1,24 @@
 import { Client, ClientOptions } from "@elastic/elasticsearch";
 import { getSecret } from "./aws";
+import log from "./logging";
 
 type ClientParameters = {
   pipelineDate: string;
   serviceName: string;
+  hostEndpointAccess: "private" | "public";
 };
 
 const getElasticClientConfig = async ({
   pipelineDate,
   serviceName,
+  hostEndpointAccess = "public",
 }: ClientParameters): Promise<ClientOptions> => {
   const secretPrefix = `elasticsearch/content-${pipelineDate}`;
+  if (hostEndpointAccess === "private") {
+    log.info("Creating ES client for PrivateLink endpoint");
+  }
```

Comment:
Is it worth adding a log line for the public endpoint, so we'll know if it's doing the wrong thing from a log that's there, not a log that isn't?

Reply:
Yes